src/input_formatter/text_formatter.test.tsx

```diff
@@ -105,9 +112,9 @@ test("splitting full text into readable prolog format including formatted expres
 		"abs List: \n \n \n Here is also an expression: $[AUNDbracketLEFTB<-->]-> D ODERNOT E]$ " +
 		"And a second one: $[5 ADD 12 equal 3 mal 5 plus 2]$");
 	const expectedResult = "[[hello,this,is,a,test],[is,it,working]," +
-		"[i,hope,so],[abs],[paragraphs,are,marked,with,an,abs,list:]," +
-		"[abs],[abs],[abs],[here,is,also,an,expression:],[[a,and,[b,<->],->," +
-		"d,or,neg,e]],[and,a,second,one:],[[5,+,12,=,3,*,5,+,2]]].";
+		"[i,hope,so],[abs],[paragraphs,are,marked,with,an,abs,list]," +
+		"[abs],[abs],[abs],[here,is,also,an,expression],[[a,and,[b,<->],->," +
+		"d,or,neg,e]],[and,a,second,one],[[5,+,12,=,3,*,5,+,2]]].";
```

Comment:
I realized it just now. The string will always end with a `.` which is because I specified it needs to end on that for the SWI-Server. But I think this will be better put into the code that makes a request to the SWI Server. I will add this later, but this can be put away for now.

Reply:
okay i will remove adding the dot at the end